{{cookiecutter.project_slug}}/README.rst

```diff
@@ -6,6 +6,52 @@ Introduction
 Usage, etc.
 
 
+Development
+-----------
+
+This project makes use of `Pipenv <https://docs.pipenv.org/>`_., which creates
+a project virtual environment. We like to keep this virtual environment inside
+``{{ cookiecutter.project_slug }}/.venv``. This is not the default Pipenv
+behaviour, so we need to set the following environment variable:
+``export PIPENV_VENV_IN_PROJECT=1``. If you add that to your ``.bashrc``, you
+don't need to specify it each time.
+
+Install the environment::
+
+    $ cd {{ cookiecutter.project_slug }}
+    $ pipenv install --deploy --dev
+    $ mkdir -p var/static var/media var/log
+
+
+Define appropriate local settings (deltaportaal/localsettings.py):
```

Comment:
If you need `localsettings.py` in development, something is wrong. The db HOST is already defined as `db` normally and the name and so is also already defined in the docker compose file.

Reply:
Yes this is a mistake, it should be in the custom docker setup